modules/nw-networkpolicy-audit-concept.adoc

```diff
@@ -6,8 +6,6 @@
 [id="nw-networkpolicy-audit-concept_{context}"]
 = Audit logging
 
-The OVN-Kubernetes network plugin uses Open Virtual Network (OVN) ACLs to manage egress firewalls and network policies. Audit logging exposes allow and deny ACL events.
-
```

Comment:
this line: https://github.com/openshift/openshift-docs/pull/75310/files#diff-8aaae2b90131866357c6bacf9a7c1b57e2352d8ba1e80558b9e0f1a50baa85daR12
i.e 
`Audit logging is enabled per namespace by annotating the namespace with the `k8s.ovn.org/acl-logging` key as in the following example:` is true only for networkpolicy/efw so if you put it in the generic audit logging section it could get confusing. I'd suggest to take a look at netpol and efw sections to see the overlapping difference and move things there OR clarify this point as "an example and for each feature level look at how to enable this under that section kinda thing"